### PR TITLE
feat: Add api for listing orgs and convert orgs cmd

### LINKED ISSUE #239: Implement `org` cli command through the api
# Acceptance Criteria

- command `create-org` becomes `orgs create`
- `orgs create` -> `POST api/v1/orgs`
- `orgs` -> `GET api/v1/orgs`

### PR DESCRIPTION
**Pull request by andreas-kupries**
Fixes #239

### CODE REVIEW

**Comment by jimmykarily** [comment]
I see we use both `organisation` and `organization` spellings in code. I suggest we stick to one (the `z` version?) to avoid confusion in editor auto-completion and searches in code.

**Comment by andreas-kupries** [comment]
Huh. Sure. ... We might have to rename the controller file then. IIRC it is one of the things using `s` spelling.

**Comment by rohitsakala** [comment]
> Huh. Sure. ... We might have to rename the controller file then. IIRC it is one of the things using `s` spelling.

Fixed.

File: acceptance/api_v1_applications_test.go [hunk 1]
```diff
@@ -34,139 +27,250 @@ var _ = Describe("API Application Endpoints", func() {
...
+				bodyBytes, err := ioutil.ReadAll(response.Body)
+				Expect(err).ToNot(HaveOccurred())
+				Expect(response.StatusCode).To(Equal(http.StatusNotFound), string(bodyBytes))
+			})
 		})
 	})
 
-	Describe("DELETE api/v1/orgs/:org/applications/:app", func() {
-		var service string
+	Context("Orgs", func() {
```
Comment (jimmykarily) [inline acceptance/api_v1_applications_test.go:178]: I think this belongs to another test file. This one is called `api_v1_applications_test.go` . We should better use a `api_v1_orgs_test.go` for all orgs related tests.

File: acceptance/api_v1_applications_test.go [hunk 2]
```diff
@@ -34,139 +27,250 @@ var _ = Describe("API Application Endpoints", func() {
...
+				bodyBytes, err := ioutil.ReadAll(response.Body)
+				Expect(err).ToNot(HaveOccurred())
+				Expect(response.StatusCode).To(Equal(http.StatusBadRequest), string(bodyBytes))
+				Expect(string(bodyBytes)).To(Equal("Name of organization to create not found\n"))
+			})
+
+			It("fails for a known organization", func() {
+				// Create the org
+
+				response, err := Curl("POST", fmt.Sprintf("%s/api/v1/orgs", serverURL),
```
Comment (jimmykarily) [inline acceptance/api_v1_applications_test.go:239]:
```suggestion
				response, err := Curl("POST", fmt.Sprintf("%s/api/v1/orgs", serverURL),
```

File: acceptance/api_v1_applications_test.go (hunk 1 above)
Comment (andreas-kupries) [reply acceptance/api_v1_applications_test.go:178]: That bogosity is on me. Apologies.

File: acceptance/api_v1_applications_test.go (hunk 2 above)
Comment (andreas-kupries) [reply acceptance/api_v1_applications_test.go:239]: Is the difference in the whitespace ?

**Review by jimmykarily — DISMISSED** [review]

File: acceptance/api_v1_applications_test.go (hunk 2 above)
Comment (jimmykarily) [reply acceptance/api_v1_applications_test.go:239]: yes the empty line

File: acceptance/api_v1_applications_test.go (hunk 1 above)
Comment (rohitsakala) [reply acceptance/api_v1_applications_test.go]: Done.

**Review by jimmykarily — APPROVED** [review]

**Pull request merged**